### PR TITLE
[plugin.video.dailymotion_com] v2.4.2+matrix.1

### PR DESCRIPTION
**Pull request by Gujal00**
### Description
Same as #3022 but for Matrix
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.dailymotion_com/addon.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		<import addon="script.module.kodi-six" version="0.0.1"/>
+        <import addon="script.module.requests" version="2.4.3"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">	    
+        <platform>all</platform>
+	    
+        <summary lang="en">Movies, TV, Music, User-Clips, and Livestreams</summary>
```
Comment (enen92) [inline plugin.video.dailymotion_com/addon.xml]: en_GB

File: plugin.video.dailymotion_com/addon.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		<summary lang="fr">Films, Séries TV, Musique, Clips, et Live</summary>
```
Comment (enen92) [inline plugin.video.dailymotion_com/addon.xml]: fr_FR

File: plugin.video.dailymotion_com/addon.xml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">	    
+        <platform>all</platform>
+	    
+        <summary lang="en">Movies, TV, Music, User-Clips, and Livestreams</summary>
+		<summary lang="fr">Films, Séries TV, Musique, Clips, et Live</summary>    
+        <summary lang="he">סרטים, טלוויזיה, מוסיקה, סירטונים של המשתמש, תלת מימד והזרמה חיה</summary>
```
Comment (enen92) [inline plugin.video.dailymotion_com/addon.xml]: he_IL

File: plugin.video.dailymotion_com/addon.xml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">	    
+        <platform>all</platform>
+	    
+        <summary lang="en">Movies, TV, Music, User-Clips, and Livestreams</summary>
+		<summary lang="fr">Films, Séries TV, Musique, Clips, et Live</summary>    
+        <summary lang="he">סרטים, טלוויזיה, מוסיקה, סירטונים של המשתמש, תלת מימד והזרמה חיה</summary>   
+        
+		<description lang="en">Dailymotion attracts 300 million users from around the world, who watch 3.5 billion videos on its player each month.</description>
```
Comment (enen92) [inline plugin.video.dailymotion_com/addon.xml]: same for all these

File: plugin.video.dailymotion_com/addon.xml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+		<description lang="fr">Dailymotion attire 300 millions d’utilisateurs qui regardent 3,5 milliards de vidéos sur son player chaque mois, à travers le monde.</description>
+		<description lang="he">Dailymotion מושך 300 מיליון משתמשים מרחבי העולם שצופים ב -3.5 מיליארד סרטונים מדי חודש.</description>
+	
+		<disclaimer lang="en">Some parts of this addon may not be legal in your country of residence - please check with your local laws before installing.</disclaimer>
```
Comment (enen92) [inline plugin.video.dailymotion_com/addon.xml]: same for all these

File: plugin.video.dailymotion_com/addon.xml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+	    
+        <news>
+[B]Version 2.4.2 - 15 July 2020[/B] 
+ - Rewrite with six library instead of future
+ - Fix playback issues
+		</news>
+		<website>https://www.dailymotion.com</website>
+        <source>https://github.com/Gujal00/plugin.video.dailymotion_com</source>
+        <forum>https://forum.kodi.tv/showthread.php?tid=142626</forum>
+        <email>gujal at protonmail dot com</email>
```
Comment (enen92) [inline plugin.video.dailymotion_com/addon.xml:35]: license tag missing, please define it using the SPDX identifier of your license. See spdx.org for the list of identifiers

File: plugin.video.dailymotion_com/resources/settings.xml [hunk 7]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (enen92) [inline plugin.video.dailymotion_com/resources/settings.xml:1]: would be nice to migrate the settings to the new format at some point

**Review by enen92 — COMMENTED** [review]
see comments

File: plugin.video.dailymotion_com/addon.xml (hunk 2 above)
Comment (Gujal00) [reply plugin.video.dailymotion_com/addon.xml]: Done as per advice

File: plugin.video.dailymotion_com/addon.xml (hunk 1 above)
Comment (Gujal00) [reply plugin.video.dailymotion_com/addon.xml]: Done as per advice

File: plugin.video.dailymotion_com/addon.xml (hunk 3 above)
Comment (Gujal00) [reply plugin.video.dailymotion_com/addon.xml]: Done as per advice

File: plugin.video.dailymotion_com/addon.xml (hunk 4 above)
Comment (Gujal00) [reply plugin.video.dailymotion_com/addon.xml]: Done as per advice

File: plugin.video.dailymotion_com/addon.xml (hunk 5 above)
Comment (Gujal00) [reply plugin.video.dailymotion_com/addon.xml]: Done as per advice

File: plugin.video.dailymotion_com/addon.xml (hunk 6 above)
Comment (Gujal00) [reply plugin.video.dailymotion_com/addon.xml:35]: Done as per advice

File: plugin.video.dailymotion_com/resources/settings.xml (hunk 7 above)
Comment (Gujal00) [reply plugin.video.dailymotion_com/resources/settings.xml:1]: I started doing it but unfortunately noticed that the new format is not supported properly in Leia. With most users still on Leia, i am sticking to the old format for now as it is a single codebase for both Leia and Matrix on my git.
Will certainly do it if Leia nightly fixes it

**Pull request merged**